fairlearn/grid_search.py

```diff
@@ -0,0 +1,144 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""
+This module implements a 1d grid search for binary classification with a single
+binary protected attribute.
+
+This is a simplification of the full exponentiated gradient search
```

Comment:
nit: it's not really related to exponentiated gradient search, so I feel like calling it a simplification seems a bit misleading if people don't have the context of the paper. We could point to the paper section (I believe 3.4) and say that it's just there as a baseline for comparison with expgrad etc. Open to suggestions, though :-)

Reply:
'Simplified alternative' perhaps? I agree that it's not a direct simplification of `expgrad`. It can't be since each point is a single model, not an ensemble :-)

Reply:
How about we drop the reference to expgrad. It's effectively a different algorithm.

Reply:
Reference to `expgrad` dropped